src/components/templates/PartyMap/components/Map/MapRoom.tsx

```diff
@@ -3,7 +3,7 @@ import classNames from "classnames";
 
```

Comment:
I would potentially do more in the actual logic of this file when the type is unclickable, to prevent any unneeded things running/etc when that type is set.

Reply:
Probably would prevent `handleRoomHovered`/`handleRoomUnhovered` from doing anything. Or just outright don't set `onClick`/`onMouseEnter`/`onMouseLeave` (can set them to undefined or something? Maybe a ternary. Maybe in and object and just spread them into the props when not an unclickable type?)

Reply:
Should make the `selectRoom` prop optional as well.

Reply:
I'd possibly even just outright conditionally render the whole `maproom__label` block rather than hiding it with CSS/etc.